sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeActionsAsyncClient.java

```diff
@@ -249,6 +256,22 @@ private JobManifestTasks getJobManifestTasks(TextAnalyticsActions actions) {
                                 .setModelVersion(getNotNullModelVersion(action.getModelVersion()))
                         );
                         return entityLinkingTask;
+                    }).collect(Collectors.toList()))
+            .setSentimentAnalysisTasks(actions.getAnalyzeSentimentOptions() == null ? null
+                : StreamSupport.stream(actions.getAnalyzeSentimentOptions().spliterator(), false).map(
+                    action -> {
+                        if (action == null) {
+                            return null;
+                        }
+                        final SentimentAnalysisTask sentimentAnalysisTask = new SentimentAnalysisTask();
+                        sentimentAnalysisTask.setParameters(
+                            // TODO: currently, service does not set their default values for model version, we
+                            // temporally set the default value to 'latest' until service correct it.
+                            // https://github.com/Azure/azure-sdk-for-java/issues/17625
+                            new SentimentAnalysisTaskParameters()
```

Comment:
out of curiosity, do you know when will this be fixed?

Reply:
I checked this issue again. Service already resolved the issue. I will close this issue. Thanks Mariana

Reply:
Awesome!